src/js/pages/Values/OneValue.jsx

```diff
@@ -264,20 +267,55 @@ class OneValue extends Component {
             <FilterChoices>
               <Chip
                 key="forThisElectionKey"
-                id = "forThisElection"
+                id="forThisElection"
                 label={<span style={showEndorsersForThisElection ? { fontWeight: 600 } : {}}>For This Election</span>}
                 className={showEndorsersForThisElection ? classes.selectedChip : classes.notSelectedChip}
                 component="div"
-                onClick={() => this.changeListModeShown('voterGuidesForThisElection')}
+                // Yousra Elzamzami - WV-1135
```

Comment:
Hi @YousraElzamzami, one of the principles I use to guide me is look for ways to reuse code in a function instead of putting mostly the same code in two places. Based on that, I recommend moving your dataLayer code to be inside changeListModeShown.